NugetLightRequest.cs

```diff
@@ -1755,40 +1907,69 @@ private IEnumerable<PackageItem> SearchForPackages(PackageSource source, string
                     }
 
                     //Deal with a case when a user type Find-Package *
-                    if (String.Equals(tempName, "*", StringComparison.OrdinalIgnoreCase)) {
+                    if (String.Equals(tempName, "*", StringComparison.OrdinalIgnoreCase))
+                    {
                         //We use '' instead of "*" because searchterm='*' does not return the entire repository.
```

Comment:
>//We use '' instead of "*" because searchterm='*' does not return the entire repository. [](start = 24, length = 88)

is this still the case for V3?

Reply:
Yes - the v3 and v2 search query parsing should be the same, so it's never correct to pass in '*' (at least for now).